puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/packaging_build_rpm.groovy

```diff
@@ -0,0 +1,77 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
```

Comment:
Previously we had concurrent builds. Any reason to disable this now?

Reply:
Whoops, copy/paste I suppose.